web/beacon-app/src/features/projects/components/EditProject/EditProjectForm.tsx

```diff
@@ -47,7 +47,7 @@ function EditProjectForm({ handleSubmit, project }: EditProjectModalFormProps) {
     <FormikProvider value={formik}>
       <Form className="space-y-3">
         <TextField label={t`Current Project Name`} {...getFieldProps('project')} isDisabled />
-        <TextField label={t`New Project Name`} {...getFieldProps('name')} />
+        <TextField label={t`New Project Name (optional)`} {...getFieldProps('name')} />
```

Comment:
I noticed that the `New Project Name` font-size label is 12px while the the font-size for the `Description` is 14px. Also, should the form display `Project Description` or `New Project Description` in this modal? Sorry, for not catching that before.